do-catalog/src/utils/form-url.js

```diff
@@ -0,0 +1,7 @@
+export function formUrl(category, country, datastream) {
+  var formUrl = "https://carto.com/platform/location-data-streams/form/?";
+  formUrl = category ? formUrl + "category=" + category + "&" : formUrl;
```

Comment:
```suggestion
  formUrl += category ? `category=${category}&` : '';
```

Reply:
Same for the rest, take a look to https://github.com/CartoDB/common-ui/blob/9bfed2a286fa88b468d069251cb67ef62f9420f7/do-catalog/src/store/actions/catalog.js#L32